restcountries/__init__.py

```diff
@@ -1,4 +1,4 @@
 # simpler import as described in the readme
-from restcountries.base import RestCountryApiV2
+from restcountries.base import Country, RestCountryApiV2
```

Comment:
Usually this class is not needed for imports - why add it here?

Reply:
This allows for something like 
```python
countries: list[Country] = rapi.get_all()
print(countries[0].alpha_code)
```
in order to pass pyright or a similar type checker, as said checker can now verify that alpha_code is a field that the first object of that list is guaranteed/supposed to have.